core/dbt/parser/schema_yaml_readers.py

```diff
@@ -228,19 +228,11 @@ def _get_input_measures(
     def _get_period_agg(self, unparsed_period_agg: str) -> PeriodAggregation:
         return PeriodAggregation(unparsed_period_agg)
 
-    def _get_optional_grain_to_date(
-        self, unparsed_grain_to_date: Optional[str]
-    ) -> Optional[TimeGranularity]:
-        if not unparsed_grain_to_date:
-            return None
-
-        return TimeGranularity(unparsed_grain_to_date)
-
     def _get_optional_time_window(
         self, unparsed_window: Optional[str]
     ) -> Optional[MetricTimeWindow]:
         if unparsed_window is not None:
-            parts = unparsed_window.split(" ")
+            parts = unparsed_window.lower().split(" ")
```

Comment:
Does DSI/Metricflow _require_ these be lowercase now? If so, this is breaking as for already parsed manifests 🤔

Reply:
Nope! It should not be case sensitive. This is lowercasing it before we get it so that the user doesn't need to worry about lowercasing it but we can expect lowercase on our side. Will added tests for case sensitivity in DSI, too.

Reply:
The words that concern me are
> but we can expect lowercase on our side
I don't think you can unfortunately, but I could be wrong about that. Could you point me to the tests around case sensitivity in DSI?

Reply:
Yep!
https://github.com/dbt-labs/dbt-semantic-interfaces/blob/f3a50f02b962ce7193b801784740e244057ddb69/tests/parsing/test_metric_parsing.py#L215-L216
and
https://github.com/dbt-labs/dbt-semantic-interfaces/blob/f3a50f02b962ce7193b801784740e244057ddb69/tests/parsing/test_metric_parsing.py#L240-L241

Reply:
Let me go back and make sure all other fields have this covered, too

Reply:
Alright @QMalcolm - sorry for the delay on this, had some other high prios and then didn't want to mess with code freeze! 🥶 

We bumped the version of DSI one more time with some changes to fix the issues with case sensitivity. [This is the PR](https://github.com/dbt-labs/dbt-semantic-interfaces/pull/370) with those changes, but these commits are most relevant: [1](2083a249889252823b097098cb03471f2db0d00e), [2](00abab69a8b4537abcc0ea9e544547e99a3c27a2).
Let us know if you think there are any other issues here! The thorough review was much appreciated 🙏 
Once approved, we shouldn't merge quite yet - just because we want to coordinate the merge timing so that we will have time to update mantle between when this merges and the next sync to mantle.

cc @WilliamDee

Reply:
I think github screwed up those links somehow. They point to https://github.com/dbt-labs/dbt-core/pull/2083a249889252823b097098cb03471f2db0d00e and https://github.com/dbt-labs/dbt-core/pull/00abab69a8b4537abcc0ea9e544547e99a3c27a2. Which are commits, but on DSI, not core.

Reply:
For the public observer, the correct links are
1. https://github.com/dbt-labs/dbt-semantic-interfaces/pull/370/commits/2083a249889252823b097098cb03471f2db0d00e
2. https://github.com/dbt-labs/dbt-semantic-interfaces/pull/370/commits/00abab69a8b4537abcc0ea9e544547e99a3c27a2